velox/docs/configs.rst

```diff
@@ -697,6 +697,13 @@ Spark-specific Configuration
      - integer
      -
      - The current task's Spark partition ID. It's set by the query engine (Spark) prior to task execution.
+   * - spark.legacy_time_parser_policy
```

Comment:
Do we need '_policy' here?

Why 'parser'? Should it be 'legacy_date_formatter' or  'legacy_datetime_formatter'?

Reply:
Thanks. I changed it to 'legacy_date_formatter'.